src/configureStore.js

```diff
@@ -1,4 +1,4 @@
-import { createStore, applyMiddleware, compose, combineReducers } from 'redux'
+import { createStore, applyMiddleware, compose } from 'redux'
```

Comment:
I don't see why this revision needs to occur within this propType work. Is this related? If not, please restore to the master version.

Reply:
combineReducers isn't used, resulting in linter errors.  Reverted to master to help clarify this PR.

Reply:
Actually, someone removed combineReducers already from master.  Not sure why this is showing us as a difference.